### PR TITLE
Tidy up

### PR DESCRIPTION
**Pull request by lbodor**

### CODE REVIEW

File: src/client/app/shared/global/user-auth.service.ts [hunk 1]
```diff
@@ -23,9 +23,9 @@ export class UserRegistration {
  */
 @Injectable()
 export class UserAuthService {
-    public userLoadedEvent: EventEmitter<User> = new EventEmitter<User>();
+    public userLoadedEvent = new EventEmitter<User | null>();
```
Comment (oehm-smith) [inline src/client/app/shared/global/user-auth.service.ts]: Perhaps you need to make it:

```public userLoadedEvent: EventEmitter<any> = new ...`

File: src/client/app/shared/global/user-auth.service.ts [hunk 2]
```diff
@@ -118,8 +113,9 @@ export class UserAuthService {
     }
 
     private addEventHandlers() {
-        this.userManager.events.addUserUnloaded((e) => {
+        this.userManager.events.addUserUnloaded(() => {
             this.currentUser = null;
+            this.userLoadedEvent.emit(null);
```
Comment (oehm-smith) [inline src/client/app/shared/global/user-auth.service.ts]: Was it meant to be `emit(e)`?

File: src/client/app/shared/global/user-auth.service.ts (hunk 1 above)
Comment (lbodor) [inline src/client/app/shared/global/user-auth.service.ts]: I think that the compiler can infer that.

File: src/client/app/shared/global/user-auth.service.ts (hunk 2 above)
Comment (lbodor) [inline src/client/app/shared/global/user-auth.service.ts]: I don't think that there is an `e`.

**Pull request merged**